### PR TITLE
Enable heated seat and steering wheel based on capability of vehicle

### PR DESCRIPTION
**Pull request by ehendrix23**
This PR will enable heated seats and heated steering wheel entities based on the capability of the vehicle.
This allows those entities initially to be created as disabled, but first time climate data is received those that are available are then enabled.
Enabling is only checked once per startup once data has been received. Once an entity is enabled is will stay enabled.
An entity that was disabled by the user will not be re-enabled again.

If approved then I plan to update teslajsonpy to make all heated seats and the heated steering wheel switch as disabled by default. 
Note, that anyone who already has those entities now will not see any change since they are then already enabled. This will only be seen when newly adding the component.

### CODE REVIEW

**Comment by alandtse** [comment]
Just so we avoid deadlock, I'm assuming you need to bump teslajsonpy so this isn't ready for integration. Let me know if I'm wrong. I try to do releases on weekends in case that helps your planning. No rush.

**Comment by ehendrix23** [comment]
> Just so we avoid deadlock, I'm assuming you need to bump teslajsonpy so this isn't ready for integration. Let me know if I'm wrong. I try to do releases on weekends in case that helps your planning. No rush.

Yep, just bumped teslajsonpy version and updated other files for new version of this item unless I missed something.
Also prepped changelog but commit still has to be updated since this hasn't merged yet. Do check make sure it looks good. :-)

**Comment by alandtse** [comment]
Please note that changelogs are autogenerated by the commit message.

**Comment by ehendrix23** [comment]
> Please note that changelogs are autogenerated by the commit message.

Do I need to push a new commit to revert that then?

**Comment by alandtse** [comment]
Yes. Otherwise iti'll just have to entries for 2.1.2

File: pyproject.toml [hunk 1]
```diff
@@ -1,13 +1,13 @@
 [tool.poetry]
 name = "tesla"
-version = "2.1.1"
+version = "2.1.2"
```
Comment (alandtse) [inline pyproject.toml:3]: Same here.

File: pyproject.toml [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 description = "A fork of the Home Assistant tesla integration using a oauth proxy to login."
 authors = ["Alan D. Tse <alandtse@gmail.com>"]
 license = "Apache-2.0"
 
 [tool.poetry.dependencies]
 python = "^3.9"
-teslajsonpy = "^2.0.3"
+teslajsonpy = "^2.1.0"
```
Comment (alandtse) [inline pyproject.toml:10]: This is good to bump.

File: pyproject.toml (hunk 1 above)
Comment (ehendrix23) [reply pyproject.toml]: Reverted

File: custom_components/tesla_custom/manifest.json [hunk 3]
```diff
@@ -22,5 +22,5 @@
     }
   ],
   "iot_class": "cloud_polling",
-  "version": "2.1.1"
+  "version": "2.1.2"
```
[the remark this answers is not in the document]
Comment (ehendrix23) [reply custom_components/tesla_custom/manifest.json]: One_sap922

[the remark this answers is not in the document]
Comment (ehendrix23) [reply custom_components/tesla_custom/manifest.json]: Reverted

**Pull request merged**